FastSurferCNN/README.md

```diff
@@ -131,6 +131,21 @@ python3 generate_hdf5.py \
 
 ```
 
+#### Example Command Sagittal using --data_dir instead of --csv_file
+--data_dir specifies the path in which the data is located, with --pattern we can select subjects from the specified path. By default the pattern is "*" meaning all subjects will be selected.
+In the following example we selecting subject1 till subject19.
```

Comment:
... we select subject1 till subject9.

Reply:
Change this line to subject10 to subject19 (which is what we select with --pattern "subject1[0-9]".